libs/src/api/File.ts

```diff
@@ -38,6 +38,28 @@ export class File extends Base {
     this.ServiceProvider = serviceProvider
   }
 
+  /**
+   * Fetches a file from Discovery Provider with a given CID.
+   * @param cid IPFS content identifier
+   * @param responseType axios response type
+   */
+  async fetchCIDFromDiscovery(
+    cid: string,
+    responseType: ResponseType = 'blob'
```

Comment:
this defaults to a blob type and then later it defaults to json?

Reply:
whoops. changed both to json https://github.com/AudiusProject/audius-protocol/pull/5059/commits/10f502f4ac491da7fef1115ade6d76dc5b6385c7

had as blob default to match the legacy libs fetchCID below but every callsite passes in responseType json